charts/dshackle/templates/service.yaml

```diff
@@ -4,11 +4,13 @@ metadata:
   name: {{ include "dshackle.fullname" . }}
   labels:
     {{- include "dshackle.labels" . | nindent 4 }}
+  annotations:
+    {{ toYaml .Values.service.annotations | nindent 4 }}
 spec:
   type: {{ .Values.service.type }}
   ports:
-    - port: {{ .Values.httpPort }}
-      targetPort: http
+    - port: {{ .Values.externalHttpPort }}
+      targetPort: {{ .Values.internalHttpPort }}
```

Comment:
@cristidas I think if we plan to open PR to back-merge it we should make as less changes as possible. I would suggest to introduce `externalHttpPort` but keep using `httpPort` as internal one

Reply:
@OleksandrUA it wouldn't be all these changes in a PR, and I've already opened one only for the changes related to `service.type: LoadBalancer`. We'll see after that if we'll need to refactor the changes for the ports and create a PR to upstream